app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class MessagesController < ApplicationController
+      def index
+        greetings = Message.find(rand(1..5))
```

Comment:
- [ ] Your frontend is connecting properly to the API backend, however, not able to generate random message as required, as there appear to be no random message found. You can seed some random greetings in the `#db\seeds.rb` file so as to populate the database with some random greetings. Kindly ensure to improve upon the same.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/197375511-96173f22-c6e1-47dd-8b1b-40a88e34d21f.png)